src/main/java/com/kodilla/ecommercee/domain/CartDto.java

```diff
@@ -1,22 +1,17 @@
 package com.kodilla.ecommercee.domain;
 
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+
 import java.util.ArrayList;
```

Comment:
Tutaj większość importów nie działa, wiem, że to czepialstwo ale wystarczy użyć "Ctrl+Alt+O" i czyści niepotrzebne importy w każdej klasie :)